dvc/repo/experiments/ls.py

```diff
@@ -39,6 +43,11 @@ def ls(
         name = baseline[:7]
         if tags[baseline] or ref_heads[baseline]:
             name = tags[baseline] or ref_heads[baseline][len(base) + 1 :]
-        results[name] = [info.name for info in ref_info_dict[baseline]]
+        for info in ref_info_dict[baseline]:
+            if git_remote:
+                exp_rev = None
+            else:
+                exp_rev = next(repo.scm.branch_revs(str(info), info.baseline_sha))[:7]
```

Comment:
Not sure if there's a better way to get the rev. Took this from: https://github.com/iterative/dvc/blob/9c9ca39fb4cb6718e47b2650e0c2da47445bca2e/dvc/repo/experiments/utils.py#L216